### ISSUE TITLE
Option to change the Gui Scale

### ISSUE THREAD

**Issue by PossiblyRemember** [studio-minus/ppg-bugs-and-requests#1656]
**Feature description**
Makes the menus smaller or larger with a slider.

**Feature purpose**
More space to do stuff without large menus in the way

**Additional context**
None

**Comment by DindinYT37** [comment]
Agree. Even though this isn't zooi's problem as this is caused by mods - it would probably also solve this issue:
![image](https://user-images.githubusercontent.com/64973205/178114314-59018a96-8299-4e87-aa2c-c82d22a4101f.png)
